mockup-contributions_list.php

```diff
@@ -0,0 +1,922 @@
+<?php
```

Comment:
Yep, please with that. In which case, if that's the route we're going to go down, folders will be essential, on top of the there should be a 'dev' folder for any new SCSS files so I know when producing builds what's not completed.